Assets/MixedRealityToolkit.SDK/Features/Utilities/PointerUtils.cs

```diff
@@ -144,6 +146,131 @@ public static IEnumerable<IMixedRealityPointer> GetPointers()
                 }
             }
         }
-    }
 
-}
\ No newline at end of file
+        /// <summary>
+        /// Queries input system for the behavior of a given pointer type. See <seealso cref="Microsoft.MixedReality.Toolkit.Input.PointerBehavior"/>.
+        /// </summary>
+        /// <typeparam name="T">Type of pointer to query</typeparam>
+        /// <param name="handedness">Handedness to query</param>
+        /// <returns><seealso cref="Microsoft.MixedReality.Toolkit.Input.PointerBehavior"/> for the given pointer type and handedness</returns>
+        public static PointerBehavior GetPointerBehavior<T>(Handedness handedness) where T : class, IMixedRealityPointerHandler
+        {
+            return GetPointerBehavior(typeof(T), handedness);
+        }
+
+        /// <summary>
+        /// Queries input system for behavior of given pointer type and handedndess
+        /// </summary>
+        /// <typeparam name="T">Type of pointer to query</typeparam>
+        /// <param name="handedness">Handedness to query</param>
+        /// <returns><seealso cref="Microsoft.MixedReality.Toolkit.Input.PointerBehavior"/> for the given pointer type and handedness</returns>
+        public static PointerBehavior GetPointerBehavior(Type type, Handedness handedness)
+        {
+            if (CoreServices.InputSystem.FocusProvider is FocusProvider focusProvider)
+            {
+                if (type == typeof(GGVPointer))
+                {
+                    return focusProvider.GazePointerBehavior;
+                }
+                return focusProvider.GetPointerBehavior(type, handedness);
+            }
+            else
+            {
+                WarnAboutSettingCustomPointerBehaviors();
+                return PointerBehavior.Default;
+            }
+        }
+
+        /// <summary>
+        /// Sets the behavior for the hand ray with given handedness
+        /// </summary>
+        /// <param name="pointerBehavior">Desired <seealso cref="Microsoft.MixedReality.Toolkit.Input.PointerBehavior"/>.</param>
+        /// <param name="handedness">Specify handedness to restrict to only right, left hands.</param>
+        public static void SetHandRayPointerBehavior(PointerBehavior pointerBehavior, Handedness handedness = Handedness.Any)
+        {
+            SetPointerBehavior<LinePointer>(pointerBehavior, handedness);
+        }
+
+        /// <summary>
+        /// Sets the behavior for the motion controller ray with given handedness
+        /// </summary>
+        /// <param name="pointerBehavior">Desired <seealso cref="Microsoft.MixedReality.Toolkit.Input.PointerBehavior"/>.</param>
+        /// <param name="handedness">Specify handedness to restrict to only right, left.</param>
+        public static void SetMotionControllerRayPointerBehavior(PointerBehavior pointerBehavior, Handedness handedness = Handedness.Any)
+        {
+            SetPointerBehavior<LinePointer>(pointerBehavior, handedness);
```

Comment:
No, motion controller pointers are hand ray pointers as well :-)

Reply:
I added a way to differentiate the hand ray and motion controller -- you specify behavior for pointer type, input type. This will allow you to turn off motion controllers but not hand rays.